### PR TITLE
Clarify how unobtrusive conventions are configured for unit testing

### PR DESCRIPTION
**Pull request by andreasohlund**
Review please

### CODE REVIEW

File: Snippets/Snippets_3/UnitTesting/Unobtrusive.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    using NServiceBus.Testing;
+
+    public class Unobtrusive
+    {
+        public class Setup
+        {
+            public Setup()
+            {
+                #region SetupConventionsForUnitTests
+                MessageConventionExtensions.IsMessageTypeAction = MyConvention;
```
Comment (SimonCropp) [inline Snippets/Snippets_3/UnitTesting/Unobtrusive.cs]: is `MyConvention` a little vague without context here?

**Comment by SimonCropp** [comment]
is this missing some .md file changes?

File: Snippets/Snippets_3/UnitTesting/Unobtrusive.cs (hunk 1 above)
Comment (andreasohlund) [inline Snippets/Snippets_3/UnitTesting/Unobtrusive.cs]: The .md (that I'll add , missed that) is linking to the doco on onobtrusive. Is that enough?

Or perhaps `MyMessageTypeConvention` ?

Comment (SimonCropp) [inline Snippets/Snippets_3/UnitTesting/Unobtrusive.cs]: i think in the context of a .md with a link it is sufficient

Comment (andreasohlund) [inline Snippets/Snippets_3/UnitTesting/Unobtrusive.cs]: Updated

File: nservicebus/testing/index.md [hunk 2]
```diff
@@ -32,6 +32,12 @@ This test says that when a message of the type `RequestMessage` is processed by
 
 <!-- import TestingSaga -->
 
+## Configuring unobtrusive message conventions
+
+If you are using [unobtrusive mode](/nservicebus/messaging/unobtrusive-mode.md) you need to configure the unit test support with those conventions as shown below.
+
+<!-- import SetupConventionsForUnitTests -->s
```
Comment (SimonCropp) [inline nservicebus/testing/index.md]: s?

**Pull request merged**